src/main/java/workers/ProfessorRatingScraper.java

```diff
@@ -0,0 +1,54 @@
+package workers;
+
+import java.io.IOException;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.*;
+import java.util.HashMap;
+import java.util.Map;
+
+class ProfessorRatingScraper {
+    private static Map<String, Double> previousScores = new HashMap<String, Double>();
+
+    public static double getRating(String firstName, String lastName) {
```

Comment:
chris wants us to use static less to show we understand OOP. Can you refactor this so it's not static?